src/components/CustomCard/CustomCard.js

```diff
@@ -15,7 +15,7 @@ import {
 
 const CustomCard = ({ href, title, content, theme, decorator }) => {
   const customCardInnerClasses = cx(customCardInner, {
-    [customCardDark]: theme === 'dark',
+    [customCardDark]: theme === 'light',
```

Comment:
What I tried to achieve is to switch cards to the light theme (it works, but I think it is incorrect way of doing it).